src/eei.cpp

```diff
@@ -1001,7 +1007,7 @@ namespace hera {
   /*
    * Utilities
    */
-  void EthereumInterface::safeChargeDataCopy(uint32_t length, uint32_t baseCost) {
+  void EthereumInterface::safeChargeDataCopy(uint32_t length, unsigned baseCost) {
```

Comment:
Why have you changed this?
From my little experience I recommend using `int` or `int64_t` for cost constants as well.

Reply:
The constants are currently defined as `unsigned`, wanted to ensure it is the same here.

Reply:
:+1: